crates/bevy_render/src/mesh/primitives/circle_iterator.rs

```diff
@@ -0,0 +1,80 @@
+use bevy_math::Vec2;
+use std::f32::consts::TAU;
+
+#[derive(Debug, Clone)]
+pub(crate) struct CircleIterator {
+    count: usize,
+    theta: f32,
+    wrap: bool,
+}
+
+impl CircleIterator {
+    pub(crate) fn new(count: usize, wrap: bool) -> CircleIterator {
+        if wrap {
```

Comment:
I think this `if` could potentially be removed by `count: count + wrap as usize`.

Reply:
Correct. I received some similar feedback earlier and will revise accordingly.